include/utils.h

```diff
@@ -109,14 +109,14 @@ typedef uint8_t  _u8;
 typedef int8_t   _s8;
 inline void      open_file_to_write(std::ofstream&     writer,
                                     const std::string& filename) {
-  writer.exceptions(std::ofstream::failbit | std::ofstream::badbit);
-  if (!file_exists(filename))
+       writer.exceptions(std::ofstream::failbit | std::ofstream::badbit);
+       if (!file_exists(filename))
     writer.open(filename, std::ios::binary | std::ios::out);
   else
     writer.open(filename, std::ios::binary | std::ios::in | std::ios::out);
 
   if (writer.fail()) {
-    char buff[1024];
+         char buff[1024];
```

Comment:
This indentation looks odd!

Reply:
I absolutely agree! Is it worth revisiting our _clang-format rules and try to track down just what might have caused it to (seemingly) triple indent?

Reply:
I don't know if this is a rule issue, or a bug in the IDE's interpretation of the rule. What happens if you clang-format this file from the command line in linux? Do you see this behavior?

Reply:
@harsha-simhadri I only use linux for DiskANN, I haven't even gotten it to build on Windows yet - so I would absolutely suspect this is the _clang-format rule (and/or whatever "reasonable default" it has, if our rule file says nothing about it)